cmd/init.go

```diff
@@ -0,0 +1,61 @@
+package cmd
+
+import (
+	"bufio"
+	"os"
+
+	"github.com/BurntSushi/toml"
+	"github.com/hellofresh/klepto/pkg/config"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+// NewInitCmd creates a new init command
+func NewInitCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "init",
+		Short: "Create a fresh config file",
+		Run: func(cmd *cobra.Command, args []string) {
+			RunInit()
+		},
+	}
+
+	return cmd
+}
+
+// RunInit runs the init command
+func RunInit() {
+	log.Infof("Initializing %s", configFileName)
+
+	_, err := os.Stat(configFileName)
+	if !os.IsNotExist(err) {
+		log.Fatal("Config file already exists, refusing to overwrite")
```

Comment:
`failOnError(err, "Config file already exists, refusing to overwrite")`

Reply:
Why? This is the same thing but I don't need to check the error twice. If I use `failOnError` it will do an unnecessary check of `err != nil` which is never true.